frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -60,6 +60,15 @@ interface NewRunParametersProps {
   setIsValidInput?: (isValid: boolean) => void;
 }
 
+const protoMap = new Map<string, string>([
+  ['NUMBER_DOUBLE', 'double'],
+  ['NUMBER_INTEGER', 'integer'],
+  ['STRING', 'string'],
+  ['BOOLEAN', 'boolean'],
+  ['LIST', 'list'],
+  ['STRUCT', 'map / dict'],
```

Comment:
NIT: I am wondering what is the context of these types? Are we referring to them as Python class?

Reply:
I think we are referring to python class!